src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1127,7 +1127,7 @@ public List<LimitViolationInfos> getOverloadedLines(UUID studyUuid, UUID nodeUui
         Network network = networkStoreService.getNetwork(networkUuid, PreloadingStrategy.COLLECTION, networkModificationTreeService.getVariantId(nodeUuid));
         List<LimitViolation> violations = Security.checkLimits(network, limitReduction);
         return violations.stream()
-            .filter(v -> v.getLimitType() == LimitViolationType.CURRENT && network.getLine(v.getSubjectId()) != null)
+            .filter(v -> v.getLimitType() == LimitViolationType.CURRENT)
```

Comment:
this is the correct modification.
But I think we have to rename things a little ... everywhere we talk about "overloaded lines".
For information, Powsybl Line and TwoWindingsTransformers classes have a common ancestor: Branch.

If you agree, I suggest to rename:
- this function: getOverloadedLines -> getCurrentLimitViolations(). this avoid to mention the type (line, 2wt or branch)
- the end-point : overloaded-lines -> current-limit-violations

Also adapt other places, like comments or unit tests.

This is will change the Front too (cf rest-api.js)

Reply:
Done